File: DXCAir-API/DCXAir-API/DCXAir-API/Models/Flight.cs
namespace DCXAir_API.Models
{
    /// <summary>
    ///  Clase que almacena la información de un vuelo
    /// </summary>
    public class Flight
    {
        //Determina el origen del vuelo
        public string Origin { get; set; }

        //Determina el destino del vuelo
        public string Destination { get; set; }

        //Determina el precio del vuelo
        public string Price { get; set; }

        //Determina el transporte en el que se realizará el vuelo
        public Transport Transport { get; set; }
    }
}

File: DXCAir-web/proyecto-dxcair-web/src/app/DTOs/flight.ts
import { Transport } from "./transport";
/**
 * Clase que determina la información del vuelo
 */
export class Flight{
    //Origen del vuelo
    public origin : string;

    //Determina el destino del vuelo
    public destination : string;
   
    //Determina el precio del vuelo
    public price : string;
   
    //Determina el transporte del vuelo
    public transport : Transport;
}